_posts/2025-03-21-stockholm-meetup-recap.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Crystal Meetup in Stockholm
+summary: Recap of two days reflecting and sharing experiences on Crystal
+author: mverzilli
+categories: project community
+tags: [talk, 84codes, ecosystem, events]
+---
+
+<figure>
+  <img src="/assets/blog/2025/stockholmmeetupintro.jpg" alt="Christina Dahlén (84codes) welcomes the Crystal community to the Stockholm Crystal Meetup" />
```

Comment:
*issue:* Alt text should not duplicate the caption. It's better to leave it empty.
Ditto all below.